app/mailers/vote_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class VoteMailer < ApplicationMailer
+  default from: 'from@example.com'
+
+  def notification(vote)
+    @vote = vote
+    mail to: vote.review.user.email,
+      subject: 'Your review has a new vote!'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.